ocaml/idl/datamodel.ml

```diff
@@ -9209,6 +9211,61 @@ let alert =
     ()
 *)
 
+
+(** network sriov **)
+module Network_sriov = struct
+  let lifecycle = [Published, rel_kolkata, ""]
+
+  let configuration_mode = Enum ("configuration_mode",
+    [
+      "sysfs", "Configure network sriov by sysfs, do not need reboot";
+      "modprobe", "Configure network sriov by modbrope, need reboot";
+      "unknown", "Unknown mode";
+    ])
+
+  let create = call
+      ~name:"create"
+      ~doc:"Create a network-sriov based on the specific PIF, it will automatically create a logical PIF to connect the specific network."
+      ~params:[Ref _pif, "pif", "PIF on which to enable SRIOV";
+               Ref _network, "network", "Network to link SRIOV"]
```

Comment:
This is too vague. Please describe the purpose of the network to VMs.

Reply:
ok